src/components/Filter.jsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import styled from "styled-components";
+import $ from "jquery";
+
+const FilterHeader = styled.div`
+  padding: 10px 20px;
+  border-bottom: 1px solid #dbdbdb;
+  display: flex;
+  justify-content: space-between;
+`;
+
+const FilterButton = styled.button`
+  padding: 8px 20px;
+  font-size: 0.875rem;
+  border: 1px solid #cbcbcb;
+  border-radius: 50px;
+  background-color: #fff;
+  color: #6a6a6a;
+
+  &:active {
+    border-color: #f29300;
+    background-color: #f29300;
+    color: #fff;
+  }
+`;
+
+const FilterItems = styled.div`
+  display: none;
+  background-color: #f9f9f9;
+`;
+
+const FilterItemButton = styled.button`
+  width: 50%;
+  padding: 15px;
+  border: none;
+  font-size: 0.875rem;
+
+  &:active {
+    border-color: #f29300;
+    background-color: #f29300;
+    color: #fff;
+  }
+`;
+
+export default function Filter() {
+  const LocationInfoItems = () => {
+    const displayValue = $(".location-filter-items").css("display");
+    if (displayValue === "none") {
+      $(".filter-items").hide();
+      $(".location-filter-items").show();
+    } else {
+      $(".location-filter-items").hide();
+    }
```

Comment:
```suggestion
    $(".location-filter-items").toggle();
    $(".price-filter-items").hide();
    $(".category-filter-items").hide();
```
코드 자체는 이게 더 간단할 거 같아요.

local, price, category 모두 비슷한 코드를 공유하는데 중복을 제거할 수는 없을까요?

Reply:
그걸 시도해보려다가 실패했긴 했습니다. 음.. 좀더 고민해볼게요